src/physics/cam/wv_saturation.F90

```diff
@@ -259,41 +272,41 @@ end subroutine wv_sat_final
 !---------------------------------------------------------------------
 
 ! Compute saturation vapor pressure over water
-elemental function svp_water(t) result(es)
+function svp_water(t) result(es)
```

Comment:
Create a vectorized version of svp_water.

Reply:
Thanks, it is added.